synapse/models/person.py

```diff
@@ -61,7 +61,7 @@ def getBaseModels():
                 ('ps:contact', {'subof': 'guid', 'doc': 'A GUID for a contact info record'}),
 
                 ('ps:hasuser', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|user,inet:user'}),
-                ('ps:hashost', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|host,it:host'}),
+                ('ps:hashost', {'subof': 'comp', 'fields': 'person=ps:person,host=it:host'}),
```

Comment:
@invisig0th asked me to make this new form a `comp`. I'm tracking replacing all of these with `comp`, but will not do that as part of this PR.

Reply:
:+1: